TangoRosStreamer/app/src/main/java/eu/intermodalics/tango_ros_streamer/activities/RunningActivity.java

```diff
@@ -288,7 +296,19 @@ private void showSaveMapDialog() {
         FragmentManager manager = getFragmentManager();
         SaveMapDialog saveMapDialog = new SaveMapDialog();
         saveMapDialog.setStyle(DialogFragment.STYLE_NORMAL, R.style.CustomDialog);
-        saveMapDialog.show(manager, "MapNameDialog");
+        saveMapDialog.show(manager, "SaveMapDialog");
+    }
+
+    private void showLoadOccupancyGridDialog(boolean firstTry, java.util.ArrayList<java.lang.String> nameList) {
```

Comment:
If it failed to load the previous map, we want  to show an extra message in the dialog saying "Unable to load the occupancy grid, please try again with a different one.".